flow-client/src/main/resources/META-INF/resources/frontend/Connect.ts

```diff
@@ -369,3 +369,93 @@ export class ConnectClient {
     }
   }
 }
+
+export interface LoginResult {
+  token?: string;
+  error: boolean;
+  errorTitle: string;
+  errorMessage: string;
+}
+
+export const login = async (username: string, password: string): Promise<LoginResult> => {
```

Comment:
Why `export const login` instead of `export async function login`?
```suggestion
export async function login(username: string, password: string): Promise<LoginResult> {
```

Reply:
done